src/app/admin/students/students-modal-window/students-modal-window.component.ts

```diff
@@ -134,6 +136,14 @@ export class StudentsModalWindowComponent implements OnInit {
     };
   }
 
+  hidePassword() {
+    if (this.hide === true) {
+      return this.hide = false;
+    } else {
+      return this.hide = true;
+    }
+  }
```

Comment:
I would probably suggest
 ` return this.hide = this.hide === true ? false : true;`